src/dinitctl.cc

```diff
@@ -760,27 +760,106 @@ static void print_termination_details(int exit_status)
     }
 }
 
-// Wait for a service to reached stopped (do_stop == true) or started (do_stop == false) state.
-// Returns 0 if the service started/stopped, 1 if start/stop was cancelled or failed.
-static int wait_service_state(int socknum, cpbuffer_t &rbuffer, handle_t handle,
+// Process a SERVICEEVENT packet if it is related to the specified service handle, and optionally report
+// the service status to the user (verbose == true).
```

Comment:
This line, though, is short. The next sentence shouldn't be on a new line. That isn't done anywhere else, please stick to conventions that are in use.

Reply:
My idea was a new line for each but looks like it's better to be combined.